src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -667,6 +682,18 @@ function CreatePipelineRun(props) {
               setState(state => ({ ...state, timeout: value }))
             }
           />
+          <Toggle
+            defaultToggled={false}
+            id="pending-pipeline-toggle"
+            labelText={intl.formatMessage({
+              id: 'dashboard.createPipelineRun.pendingPipilinRun',
+              defaultMessage: 'Create PipelineRun in pending state'
+            })}
+            onChange={handleToggleChange}
+            labelA="disabled"
+            labelB="enabled"
+            aria-label="Pending PipelineRun state"
```

Comment:
These should all be translated, and let's go with 'Enabled' and 'Disabled' for consistency with other labels in the UI. I spotted a few places where our aria-labels aren't currently translated but we can do a separate scrub of those.

Do we actually need to provide an `aria-label` here or is `labelText` sufficient?

Reply:
I removed the `aria-label `, I was unsure if both we needed, some of the carbon examples have both and others just have labeText, so we I guess we're fine to go to with that